nautobot/extras/models/secrets.py

```diff
@@ -46,6 +46,11 @@ class Meta:
     def __str__(self):
         return self.name
 
+    def get_provider_display(self):
+        if provider := registry["secrets_providers"].get(self.provider):
```

Comment:
For my own edification, What does `:=` do in this context?

Reply:
Walrus operator - https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions

Basically a shorter way to say:

```py
provider = registry["secrets_providers"].get(self.provider)
if provider:
```